javascript/features/killtime/killtime_manager.js

```diff
@@ -24,19 +25,28 @@ class KilltimeManager {
             'OnPlayerDeath',  // { playerid, killerid, reason }
         ].forEach(name =>
             this.callbacks_.addEventListener(toEventName(name), this.__proto__[toMethodName(name)].bind(this)));
+        this.callbacks_.addEventListener(
```

Comment:
You could also add this whole call to the list of line 25, then everything is handled for you.

Reply:
Do you mind if I change the OnPlayerDeath one to the way I did this?

The OnPlayerDeath is quite hard to read in my opinion. 
Readability > less code